src/Adapter/Shop/Repository/ShopGroupRepository.php

```diff
@@ -49,4 +128,24 @@ public function assertShopGroupExists(ShopGroupId $shopGroupId): void
             ShopGroupNotFoundException::class
         );
     }
+
+    /**
+     * @param ShopGroupId $shopGroupId
+     *
+     * @return ShopId[]
+     */
+    public function getShopsFromGroup(ShopGroupId $shopGroupId): array
+    {
+        $qb = $this->connection->createQueryBuilder();
```

Comment:
Not a big deal but why building query in two times instead of one et pass the result directly to the array map? :)